docs/user-guide/getting-started/install.md

```diff
@@ -332,7 +332,18 @@ Step 4: Add the Solidity compiler (`solc`) executable's folder to your `PATH`
     ```
 </details>
 
-Step 5 (for VS Code users): Install the Certora IDE Extension
+Step 5 (optional, for Vyper users): Install the Vyper compiler (`vyper`)
+--------------------------------------------------------------------------------
+
+[Vyper](https://github.com/vyperlang/vyper) is an EVM compatible Pythonic smart contract language.
+Since the Certora Prover operates on the bytecode, it can be applied to any source-level language
+that compiles to EVM bytecode.
+
+We recommend to install Vyper either from PyPi (i.e., `pip install vyper`) or to get a 
+binary executable for the desired version.
+
+
+Step 6 (for VS Code users): Install the Certora IDE Extension
```

Comment:
We no longer mention the vs code extension in our documentation as it is not maintained anymore

Reply:
I think it's the lsp that should be mentioned